README.md

```diff
@@ -117,6 +117,8 @@ var autocomplete = require('autocomplete.js');
  1. Include `autocomplete.min.js`
  1. Initialize the auto-completion menu calling the `autocomplete` function
 
+ **Warning**: `autocomplete.js` is not compatible with the latest version algoliasearch v4, therefore we highly recommend you use algoliasearch v3 as specified in the code snippet below.
```

Comment:
it says latest here, but it's not yet released. Should it maybe say "upcoming"?

Reply:
No. Let's just merge this once is released.

Reply:
Makes sense, but it could also be useful to already say now that the v0 isn't compatible

Reply:
v4 is not public yet so it's fine to postpone this warning.